src/Microsoft.OData.Edm/Csdl/CsdlXmlWriterSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlXmlWriterSettings.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// The options/setting for CSDL writer
+    /// </summary>
+    public class CsdlXmlWriterSettings
+    {
+        /// <summary>
+        /// Disables the CSDL Scale and SRID variable value from being written as variable to Variable.
+        /// </summary>
+        public EdmLibraryCompatibility UseLegacyVariableCasing { get; set; }

```

Comment:
```suggestion
        public EdmLibraryCompatibility LibraryCompatibility { get; set; }
```

Reply:
I think I'll name this `EdmLibraryCompatibility`